tasks/src/main/scala/dagr/tasks/fgbio/GroupReadsByUmi.scala

```diff
@@ -58,6 +60,8 @@ class GroupReadsByUmi(val in:  PathToBam,
     minMapQ.foreach(m => buffer.append("-m", m))
     buffer.append("-s", strategy.toString.toLowerCase)
     edits.foreach(e => buffer.append("-e", e))
+    minUmiLength.foreach(m => buffer.append("-l", m))
+    if (allowInterContig) buffer.append("-I")
```

Comment:
```suggestion
    if (allowInterContig) buffer.append("-x")
```

Reply:
https://github.com/fulcrumgenomics/fgbio/blob/4415220d79eb3c35b98206c82f0a145a757bba61/src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala#L434

Reply:
Oops, accidentally revealed the internal debate I had with myself over the flag choice